mopidy_mpd/network.py

```diff
@@ -125,7 +125,9 @@ def create_server_socket(self, host, port):
         socket_path = get_unix_socket_path(host)
         if socket_path is not None:  # host is a path so use unix socket
             sock = create_unix_socket()
+            oldmask = os.umask(0o002) # make socket writable for users in the 'mopidy' group
             sock.bind(socket_path)
+            os.umask(oldmask)
```

Comment:
The call to `sock.bind()` should be wrapped in a try/finally clause, with the second umask call being placed in the `finally` block.